src/js/App/Sidenav/Navigation.js

```diff
@@ -76,11 +76,10 @@ class Navigation extends Component {
     }
 
     render() {
-        const { settings, activeApp, navHidden, activeLocation, documentation } = this.props;
-
-        if (navHidden) {
-            document.querySelector('aside').setAttribute('hidden', true);
-        }
+        const { settings, activeApp, /*navHidden,*/ activeLocation, documentation } = this.props;
+        // if (navHidden) {
```

Comment:
Is there any reason why you removed this code?

Reply:
As far as Ryan and I can tell this code is never actually used(the specific ESI body tag is never used); because of this it was always failing any test I ran for this component. I ran the local combo of chrome/landing and removing the code did not have any effect. I commented it out rather than removing it on the off chance that it does have some bearing on some component I'm not aware of; and we can fix it back easily.

Reply:
As far as Ryan and I can tell this code is never actually used(the specific ESI body tag is never used); because of this it was always failing any test I ran for this component. I ran the local combo of chrome/landing and removing the code did not have any effect. I commented it out rather than removing it on the off chance that it does have some bearing on some component I'm not aware of; and we can fix it back easily.